### PR TITLE
Remove datetime input type ( SOFTWARE-5145 )

### PR DESCRIPTION
**Pull request by CannonLock**
Converted the datetime input into date and time inputs which are more broadly supported

I manually plugged in inputs into Chrome, Firefox, and Safari to test. 

Also confirmed wider support here -> https://caniuse.com/input-datetime

### CODE REVIEW

**Comment by brianhlin** [comment]
@edquist can you take a look? Once you're satisfied and we merge this, please cut a `v1.17.2` release to kick off a container build (see https://github.com/opensciencegrid/topology/releases/tag/v1.17.1 for details)

File: src/webapp/forms.py [hunk 1]
```diff
@@ -132,8 +132,11 @@ class GenerateResourceGroupDowntimeForm(FlaskForm):
     ])
     description = StringField("Description (the reason and/or impact of the outage)", [InputRequired()])
 
-    start_datetime = DateTimeLocalField("Local Start Datetime", format='%Y-%m-%dT%H:%M', validators=[InputRequired()])
-    end_datetime = DateTimeLocalField("Local End Datetime", format='%Y-%m-%dT%H:%M', validators=[InputRequired()])
+    start_date = DateField("Local Start Date", validators=[InputRequired()])
+    start_time = TimeField("Local Start Time", validators=[InputRequired()])
+    end_date = DateField("Local End Date", validators=[InputRequired()])
+    end_time = TimeField("Local End Time", validators=[InputRequired()])
+
```
Comment (edquist) [inline src/webapp/forms.py:139]: So do these new DateField/TimeField replacements have their own built-in formatting / validation?  (I see there is no longer a `format` parameter)

File: src/webapp/forms.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import datetime
 
 from flask_wtf import FlaskForm
-from wtforms import SelectField, SelectMultipleField, StringField, \
-    TimeField, TextAreaField, SubmitField
-from wtforms.fields.html5 import DateTimeLocalField
+from wtforms.fields import SelectField, SelectMultipleField, StringField, \
```
Comment (edquist) [inline src/webapp/forms.py:4]: Is there any difference between the imports from `wtforms` and `wtforms.fields` ?

Comment (CannonLock) [reply src/webapp/forms.py:4]: No, this can be changed back. Looking at it again the previous way is more inline with the documentation.

Comment (CannonLock) [reply src/webapp/forms.py:4]: https://wtforms.readthedocs.io/en/2.3.x/crash_course/#exploring-in-the-console

File: src/webapp/forms.py (hunk 1 above)
Comment (edquist) [reply src/webapp/forms.py:139]: >the browser provides an input ui

Cool

**Comment by edquist** [comment]
https://github.com/opensciencegrid/topology/releases/tag/v1.17.2

**Pull request merged**